### ISSUE TITLE
Anchours no longer working in space

### ISSUE THREAD

**Issue by nosknut** [StargateMC/StargateMCPublic#383]
The chunk loaders have now stopped turning on while in space. That completly erases all the work i did with the web interface. Plz get chunkloading back there asap :|

**Comment by Bysokar** [comment]
You'll have to explain this one.
Logically speaking there is no way they stopped working randomly. 

Where were you exactly? Did the chunk loader stop working after a wArp jump?

**Comment by Maxstripe** [comment]
they show red even tho theyre working. as far as i can tell theyre still operational

**Comment by Bysokar** [comment]
And the red comes on after a jump?

**Comment by nosknut** [comment]
No. It is constantly red and the miner does no longer respond to commands remotely so idk, but i thinnk it is off.
